apps/prairielearn/elements/pl-string-input/pl-string-input.mustache

```diff
@@ -1,82 +1,171 @@
+<script>
+    $(function(){
+        document.querySelectorAll('.pl-string-input-multiline#pl-string-input-{{uuid}}-area').forEach((input) => {
+            input.addEventListener('input', function () {
+            // Adjusts the height based on the feedback content. If the feedback changes, the height
+            // changes as well. This is done by resetting the height (so the scrollHeight is computed
+            // based on the minimum height) and then using the scrollHeight plus padding as the new height.
+            this.style.height = '';
+            if (this.scrollHeight) {
+                const style = window.getComputedStyle(this);
+                this.style.height =
+                this.scrollHeight + parseFloat(style.paddingTop) + parseFloat(style.paddingBottom) + 'px';
+            }
+            });
+            input.dispatchEvent(new Event('input'));
+        });
+    });
+</script>
+
+<!-- START pl-string-input -->
```

Comment:
Nit: Convert these to mustache comments so they don't contribute to the size of markup spit out by the element.

Reply:
👀 this is actually just left over from debugging and should be removed entirely.

Reply:
Comments formatted fixed here: https://github.com/PrairieLearn/PrairieLearn/pull/9438/commits/14f2b8e01fa7f8b81b0ecf90f571f9c0524b1fbe